c2rust-analyze/src/rewrite/ty.rs

```diff
@@ -325,19 +326,38 @@ impl<'a, 'tcx> intravisit::Visitor<'tcx> for HirTyVisitor<'a, 'tcx> {
     fn nested_visit_map(&mut self) -> Self::Map {
         self.acx.tcx().hir()
     }
+
+    fn visit_stmt(&mut self, s: &'tcx hir::Stmt<'tcx>) {
+        match s.kind {
+            // A local with a user type annotation
+            StmtKind::Local(hir_local) if hir_local.ty.is_some() => {
+                let rw_lcx = LabeledTyCtxt::new(self.acx.tcx());
```

Comment:
This code should use `self.rw_lcx` instead of creating a new one.

Reply:
dc598e4a